.travis.yml

```diff
@@ -4,17 +4,30 @@ go_import_path: github.com/KohlsTechnology/eunomia
 go:
 - "1.12"
 env:
-  - GO111MODULE=on
+  - GO111MODULE=on CHANGE_MINIKUBE_NONE_USER=true OPERATOR_SDK_VERSION=v0.9.0 KUBE_VERSION=v1.14.2
```

Comment:
Not familiar with travis, but is there any chance to have one ENV variable per line?

Reply:
@Smiley73 specifying env vars on separate lines creates a "matrix" for test cases, passing different sets of env vars to test different versions, settings etc. This is how it should be set.

Reply:
@Smiley73 actually, let do one refactor.